inc/namespace.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * SEO module functions.
+ *
+ * @package hm-platform/seo
+ */
+
+namespace HM\Platform\SEO;
+
+use const HM\Platform\ROOT_DIR;
+use function HM\Platform\get_config;
+use HM\Platform\Module;
+
+function bootstrap( Module $module ) {
+	$settings = $module->get_settings();
+
+	if ( $settings['redirects'] ) {
+		add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_redirects' );
+	}
+
+	if ( $settings['sitemaps'] ) {
+		add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_sitemaps' );
+	}
+
+	if ( $settings['metadata'] ) {
+		add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_metadata' );
+	}
+
+	if ( $settings['amp'] ) {
+		add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_amp' );
+	}
+
+	if ( $settings['fbia'] ) {
+		add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_instant_articles' );
+	}
+
+	// Read robots.txt file into robots.txt route handled by WP.
+	add_filter( 'robots_txt', __NAMESPACE__ . '\\robots_txt', 10 );
+}
+
+function get_bool_callback( bool $condition ) {
+	return $condition ? '__return_true' : '__return_false';
+}
+
+function load_redirects() {
+	require_once ROOT_DIR . '/vendor/humanmade/hm-redirects/hm-redirects.php';
+}
+
+function load_sitemaps() {
+	require_once ROOT_DIR . '/vendor/humanmade/msm-sitemap/msm-sitemap.php';
+}
+
+function load_metadata() {
+	require_once ROOT_DIR . '/vendor/humanmade/wp-seo/wp-seo.php';
+	require_once ROOT_DIR . '/vendor/humanmade/meta-tags/plugin.php';
+
```

Comment:
Elsewhere I have done `$config = get_config['modules']['seo']` to avoid all the duplication